docs/model_zoo.md

```diff
@@ -13,6 +13,28 @@ You can replace `https://s3.ap-northeast-2.amazonaws.com/open-mmlab` with `https
 - For fair comparison with other codebases, we report the GPU memory as the maximum value of `torch.cuda.max_memory_allocated()` for all 8 GPUs. Note that this value is usually less than what `nvidia-smi` shows.
 - We report the inference time as the total time of network forwarding and post-processing, excluding the data loading time. Results are obtained with the script [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) which computes the average time on 2000 images.
 
+## ImageNet Pretrained Models
+
+It's common to initialize from backbone models pre-trained on ImageNet classification tasks. All pre-trained  model weight links can be found at [open_mmlab](https://github.com/open-mmlab/mmcv/blob/master/mmcv/model_zoo/open_mmlab.json). The following commonly used backbone models are available:
```

Comment:
It's -> It is
tasks -> task
model weight links -> model links
The details of the commonly used backbone models in MMDetection are listed as below